test/unit/PermissionTree.test.js

```diff
@@ -0,0 +1,62 @@
+const { expect } = require('chai');
+const PermissionTree = require('../../src/PermissionTree');
+
+describe('PermissionTree Unit Test', () => {
+    before('Create an instance of the PermissionTree', function() {
```

Comment:
I also used the models of the booted service/app several times in unit tests. I'am not fully sure if this ain't an integration test.

Reply:
Well I guess you are right, as long as we do not mock it, this is an integration test